src/widget/tooltipqopengl.cpp

```diff
@@ -7,14 +7,18 @@
 #include <QToolTip>
 #include <memory>
 
-ToolTipQOpenGL::ToolTipQOpenGL() {
+#include "widget/wglwidget.h"
```

Comment:
this is already included in the form of `<widget/wglwidget.h>` on line 3... I guess remove that one then?

Reply:
Removed.